File: cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float cash;
    int i = 0;   // number of coins

    do
    {
        cash = get_float("Change in dollars: ");
    }
    while (cash < 0);

    // convert to integer
    int n = round(cash * 100);

    // 25 cent's coins
    while (n >= 25)
    {
        i++;
        n -= 25;
    }

    // 10 cent's coins
    while (n >= 10)
    {
        i++;
        n -= 10;
    }

    //5 cent's coins
    while (n >= 5)
    {
        i++;
        n -= 5;
    }

    // 1 cent's coins
    while (n >= 1)
    {
        i++;
        n -= 1;
    }

    printf("%i\n", i);
}

File: cash.py
from cs50 import get_float

# Promp user for change
while True:
    cash = get_float("Change in dollars: ")
    if cash >= 0:
        break

# convert to integer
n = round(cash * 100)

# number of 25 cents' coins
i = n // 25
# remainder
n = n % 25
# number of 10 cents' coins
i = i + n // 10
# remainder
n = n % 10
# number of 5 cents' coins
i = i + n // 5
# remainder
n = n % 5
# number of 1 cent' coins
i = i + n // 1
# remainder
n = n % 1

# Print out how many coins
print(i)
